contracts/Migrations.sol

```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.4.21 <0.7.0;
+pragma solidity >=0.5.0 <0.8.0;
```

Comment:
What's the reason for restricting to >=0.5.0 instead of >=0.4.22?  That file should work fine with 0.4.22 as best I can tell.  (It won't work with 0.4.21 because 0.4.21 doesn't have revert messages; but actually it should already have been >=0.4.22, because 0.4.21 didn't have the `constructor` keyword which the old version used, so actually 0.4.21 wouldn't have worked with the old version either.)

Reply:
Thanks @haltman-at, my solidity knowledge is lacking and that is good to know! My assumption is pet-shop will be unboxed with stable Truffle and we should stick with a Solidity version with stable (v0.5.16). I'd love to have less variations in contracts we use in our boxes eventually so that future syncing will be easier.

How would you like me to proceed with this PR?

Reply:
Don't know; up to you I guess.  Currently `bare-box` as well as its copy in `init` have `>=0.4.22`.  I figured best thing to do is to be as inclusive as possible, but, IDK, this is your area.  I'll go hit approve, and you can do it this way or you can change it.  I just wanted to make sure you were aware of what versions would work when determining what to do there!

Reply:
You've convinced me @haltman-at. The better choice is to be inclusive and sync to bare-box. Thanks for the explanations.